application/domain/domain.go

```diff
@@ -188,16 +188,16 @@ type EnvStruct struct {
 	PremiumMinimum          int `default:"25" split_words:"true"`
 
 	// PremiumFactor is multiplied by CoverageAmount to calculate the annual premium of an item
-	PremiumFactor           float64 `default:"0.02" split_words:"true"`
-	RepairThreshold         float64 `default:"0.7" split_words:"true"`
-	RepairThresholdString   string  `ignored:"true"`
-	Deductible              float64 `default:"0.05"`
-	DeductibleMinimumString string  `ignored:"true"`
-	DeductibleIncrease      float64 `default:"0.2"` // Additional deductible per strike
-	DeductibleMaximum       float64 `default:"0.45"`
-	EvacuationDeductible    float64 `default:"0.333333333" split_words:"true"`
-	StrikeLifetimeMonths    int     `default:"24" split_words:"true"`
-	VehiclePremiumFactor    float64 `default:"0.02" split_words:"true"` // TODO use actual rate
+	PremiumFactor         float64 `default:"0.02" split_words:"true"`
+	RepairThreshold       float64 `default:"0.7" split_words:"true"`
+	RepairThresholdString string  `ignored:"true"`
+	Deductible            float64 `default:"0.05"`
```

Comment:
It might be good to rename this so it's more obvious that it is a rate.  Using `envconfig` to keep value the same.

```suggestion
	DeductibleRate        float64 `envconfig:"deductible" default:"0.05"`
```

Reply:
Mainly put this here to show how it would be done, with minimum effect on env variables.  Don't strictly need this, but I feel it might be helpful.

Reply:
Ah, I didn't think of using the `envconfig` flag. Good one.